cpp/ql/src/Security/CWE/CWE-497/ExposedSystemData.ql

```diff
@@ -14,7 +14,9 @@
 
 import cpp
 import semmle.code.cpp.commons.Environment
-import semmle.code.cpp.security.OutputWrite
+import semmle.code.cpp.dataflow.TaintTracking
```

Comment:
Do we get the same results if we use semmle.code.cpp.ir.dataflow.TaintTracking?

Reply:
Ah, that fixes one of the missing results, thanks!